FBTweak/_FBEditableTweakTableViewCell.m

```diff
@@ -321,14 +326,14 @@ - (void)_updateValue:(FBTweakValue)value primary:(BOOL)primary write:(BOOL)write
     }
   } else if (_mode == _FBTweakTableViewCellModeString) {
     if (primary) {
-      _textField.text = value;
+      self.detailTextLabel.text = [value description];
```

Comment:
value is NSString, no? why do you need the description?

Reply:
fixed.